### DISCUSSION TITLE
23.11.08 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#57] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 포스트맨 에러 해결 (다음 PR)
    - 코드 리뷰 대기
- 내일은 어떤 일을 하나요?
    - 리뷰 확인 및 코드 수정
    - 테스트 코드 작성

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - executor 설정 정보 작성
  - merge 전에 test가 갑자기 실패해서 디버깅..
- 내일은 어떤 일을 하나요?
  - 디버깅 완료 후 merge
  - 은서님께 음악 작업 여쭤보고 아티스트 생성하기
  - 리뷰하기

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 3DS 허탕 ㅜㅜ
  * 민주님과 함께 디버깅
  * executor 테스트
* 내일은 어떤 일을 하나요?
  * batch job 수정
